api/jobs/api.py

```diff
@@ -440,8 +440,16 @@ def list(self, request):
             destination_soc = transition.get("soc2")
             destination_metadata = destination_soc_map.get(destination_soc)
             if destination_metadata:
+                # Fill in missing data to avoid issues downstream.
+                if destination_metadata['annual_mean_wage'] is None and destination_metadata['hourly_mean_wage'] is not None:
+                    destination_metadata['annual_mean_wage'] = destination_metadata['hourly_mean_wage'] * 52 * 40
+
```

Comment:
Looks fine to me. There is a design question re. hourly/annual wages: BLS excludes some of these wages for certain professions 

e.g. https://www.bls.gov/oes/current/oes251022.htm 

"(4) Wages for some occupations that do not generally work year-round, full time, are reported either as hourly wages or annual salaries depending on how they are typically paid."

If we decide to fill them in, should we put a note about the calculations performed if there is no corresponding BLS data point?